### PR TITLE
GEOPY-168: Empty FROM-TO on subsequent interval data

### PR DESCRIPTION
**Pull request by domfournier**
- Fix for subsequent add_data for drillhole intervals. The issue was on the reset of drillhole cells before adding the new FROM and TO. 
- Attempt to fix missing images in docs

### CODE REVIEW

**Review by sebhmg — CHANGES_REQUESTED** [review]
should be covered by tests

**Comment by domfournier** [comment]
@sebhmg Ok ready for review.

File: tests/create_drillhole_data_test.py [hunk 1]
```diff
@@ -47,43 +47,73 @@ def test_create_drillhole_data():
...
+        ), "Error with number of cells on interval data creation."
+        assert well.n_vertices == (
+            from_to_a.size + 4
+        ), "Error with number of vertices on interval data creation."
+        assert not np.any(
+            np.isnan(well.get_data("FROM")[0].values)
+        ), "FROM values not fully set."
+        assert not np.any(
+            np.isnan(well.get_data("TO")[0].values)
+        ), "TO values not fully set."
```
Comment (sebhmg) [inline tests/create_drillhole_data_test.py:89]: **question (non-blocking):** should it also assert on the expected data size for `FROM` and `TO`?

**Review by sebhmg — APPROVED** [review]

File: tests/create_drillhole_data_test.py (hunk 1 above)
Comment (domfournier) [reply tests/create_drillhole_data_test.py:89]: Added

**Pull request merged**